src/JsWebSocket.js

```diff
@@ -1,24 +1,39 @@
 /* global require exports WebSocket BROWSER_RUNTIME */
 
+const ReconnectingWebSocket = require('reconnecting-websocket');
+
+var OurWebSocket;
 if (typeof BROWSER_RUNTIME == 'undefined' || !BROWSER_RUNTIME) {
-    var OurWebSocket = require("ws");
+  OurWebSocket = require("ws");
+} else {
+  OurWebSocket = WebSocket;
 }
 
+class NoPerMessageDeflateWebSocket extends OurWebSocket {
+  constructor (url, protocols, options) {
+    options = options || {};
+    options.perMessageDeflate = false;
+    super(url, protocols, options);
+  }
+};
+
 // _mkWebsocket :: (String -> Effect Unit) -> String -> Effect WebSocket
-exports._mkWebSocket = logger => url => () => {
+exports.mkWebSocket = logger => url => () => {
```

Comment:
Applies to this whole module and the `foreign import`s in PS: why remove the leading underscore? That was our convention for FFI stuff, right?

Reply:
I thought that having a public module interface without underscores is prioritized over keeping FFI clearly separate.
I'll restore the underscores.